src/Routes/SamplePage/SamplePage.js

```diff
@@ -23,10 +22,11 @@ const SamplePage = () => {
   const [chosenImage, setChosenImage] = React.useState({
     id: undefined,
     name: undefined,
-    provider: undefined,
+    provider: AWS_PROVIDER,
```

Comment:
Does this need to have a value? What if we keep it to undefined?
I am asking because in testing, if the value is already selected, it won't perform the select operation and fail the test there. Workaround would be to add unnecessary if else block to see what is already selected 
But if nothing is selected as default,  that would work.

Reply:
Update: Nevermind. That still works.